docker-app/qfieldcloud/core/views/members_views.py

```diff
@@ -106,3 +111,6 @@ def get_object(self):
         return OrganizationMember.objects.get(
             organization=organization_obj, member=member_obj
         )
+
+    def perform_update(self, serializer):
```

Comment:
```suggestion
    def perform_update(self, serializer: ...) -> None:
```

Would you mind adding typing whereever possible?

Reply:
Added